GooglePlayInstant/Editor/PlayInstantAssetBundleUrlVerifierWindow.cs

```diff
@@ -0,0 +1,135 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Window that verifies AssetBundles from given URLs.
+    /// </summary>
+    public class PlayInstantAssetBundleUrlVerifierWindow : EditorWindow
+    {
+        private static bool _assetBundleDownloadIsSuccessful;
+        private static string _assetBundleUrl;
+        private static long _responseCode;
+        private static string _errorDescription;
+        private static double _numOfMegabytes;
+
+        private const int FieldMinWidth = 170;
+
+        /// <summary>
+        /// Creates a dialog box that details the success or failure of an AssetBundle retrieval from a given assetBundleUrl.
+        /// </summary>
+        public static void ShowWindow()
+        {
+            if (string.IsNullOrEmpty(QuickDeployConfig.Config.assetBundleUrl))
+            {
+                Debug.LogError("Problem with verifying AssetBundle: AssetBundle URL cannot be empty.");
+            }
+            else
+            {
+                // Set AssetBundle url in a private variable so that information displayed in window is consistent with
+                // the url that this was called on. 
+                _assetBundleUrl = QuickDeployConfig.Config.assetBundleUrl;
+                UpdateAssetBundleVerificationInfoWindow();
+                GetWindow(typeof(PlayInstantAssetBundleUrlVerifierWindow), true, "Play Instant AssetBundle Verify");
+            }
+        }
+
+        //TODO: Support Unity 5.6.0+
+        private static void UpdateAssetBundleVerificationInfoWindow()
+        {
+            var www = UnityWebRequestAssetBundle.GetAssetBundle(_assetBundleUrl);
+            www.SendWebRequest();
+            while (!www.isDone)
+            {
+                //TODO: implement loading bar 
+            }
+
+            var bundle = DownloadHandlerAssetBundle.GetContent(www);
+
+            _responseCode = www.responseCode;
+
+            if (www.isNetworkError || www.isHttpError)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = www.error;
+                Debug.LogErrorFormat("Problem retrieving AssetBundle from {0}: {1}", _assetBundleUrl,
+                    _errorDescription);
+            }
+            else if (bundle == null)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = "Failed to decompress data for the AssetBundle.";
```

Comment:
According to documentation "If there is an error decoding the asset bundle’s assets, the system will log the error and this property will return null."

So maybe here say "Error extracting AssetBundle. See Console log for details."